visualization/app/codeCharta/state/settings.service.ts

```diff
@@ -13,7 +22,7 @@ export interface SettingsServiceSubscriber {
 
 export class SettingsService implements FileStateServiceSubscriber {
 	private static SETTINGS_CHANGED_EVENT = "settings-changed"
-	private static DEBOUNCE_TIME = 400
+	private static DEBOUNCE_TIME = 100
```

Comment:
We should actually stick to a higher debounce time, as the sliders start lagging with larger maps. I think you reduced the debounce time, because otherwise it takes some time to open the searchPanel, but also here a separate service for storing the mode information helps (see other comment)

Reply:
I have reset that to 400ms, since we now have the SearchPanelService